modules/ossm-security-cert-manage.adoc

```diff
@@ -0,0 +1,130 @@
+    --from-file=<path>/cert-chain.pem
+----
++
+2. In the `ServiceMeshControlPlane` resource set `global.mtls.enabled` to `true` and `security.selfSigned` set to `false`. {ProductShortName} reads the certificates and key from the secret-mount files.
++
+[source,yaml]
+----
+apiVersion: maistra.io/v1
+kind: ServiceMeshControlPlane
+spec:
+  istio:
+    global:
+      mtls:
+        enabled: true
+    security:
+      selfSigned: false
+----
++
+3. To make sure the workloads add the new certificates promptly, delete the secrets generated by {ProductShortName}, named `istio.*`. In this example, `istio.default`. {ProductShortName} issues new certificates for the workloads.
++
+----
+$ oc delete secret istio.default
+----
+
+[id="ossm-cert-manage-verify-cert"]
+== Verifying your certificates
+
+Use the Bookinfo sample application to verify your certificates are mounted correctly. First, retrieve the mounted certificates. Then, verify the certificates mounted on the pod.
+
+1. Store the pod name in the variable `RATINGSPOD`.
++
+----
+$ RATINGSPOD=`oc get pods -l app=ratings -o jsonpath='{.items[0].metadata.name}'`
+----
++
+Run the following commands to retrieve the certificates mounted on the proxy.
++
+----
+$ oc exec -it $RATINGSPOD -c istio-proxy -- /bin/cat /etc/certs/root-cert.pem > /tmp/pod-root-cert.pem
+----
++
+The file `/tmp/pod-root-cert.pem` contains the root certificate propagated to the pod.
++
+----
+$ oc exec -it $RATINGSPOD -c istio-proxy -- /bin/cat /etc/certs/cert-chain.pem > /tmp/pod-cert-chain.pem
+----
++
+The file `/tmp/pod-cert-chain.pem` contains the workload certificate and the CA certificate propagated to the pod.
++
+3. Verify the root certificate is the same as the one specified by the Operator. Replace `<path>` with the path to your certificates.
++
+----
+$ openssl x509 -in <path>/root-cert.pem -text -noout > /tmp/root-cert.crt.txt
+$ openssl x509 -in /tmp/pod-root-cert.pem -text -noout > /tmp/pod-root-cert.crt.txt
+$ diff /tmp/root-cert.crt.txt /tmp/pod-root-cert.crt.txt
+----
++
+Expect the output to be empty.
++
+4. Verify the CA certificate is the same as the one specified by Operator. Replace `<path>` with the path to your certificates.
++
+----
+$ sed '0,/^-----END CERTIFICATE-----/d' /tmp/pod-cert-chain.pem > /tmp/pod-cert-chain-ca.pem
+$ openssl x509 -in <path>/ca-cert.pem -text -noout > /tmp/ca-cert.crt.txt
+$ openssl x509 -in /tmp/pod-cert-chain-ca.pem -text -noout > /tmp/pod-cert-chain-ca.crt.txt
+$ diff /tmp/ca-cert.crt.txt /tmp/pod-cert-chain-ca.crt.txt
+----
++
+Expect the output to be empty.
++
+5. Verify the certificate chain from the root certificate to the workload certificate. Replace `<path>` with the path to your certificates.
++
+----
+$ head -n 21 /tmp/pod-cert-chain.pem > /tmp/pod-cert-chain-workload.pem
+$ openssl verify -CAfile <(cat <path>/ca-cert.pem <path>/root-cert.pem) /tmp/pod-cert-chain-workload.pem
+/tmp/pod-cert-chain-workload.pem: OK
+----
+
+[id="ossm-cert-cleanup"]
+== Cleanup
```

Comment:
Can we change this heading to "Removing Certificates" or something more descriptive?